src/store/sparql.py

```diff
@@ -13,10 +17,31 @@ def setup(self):
     def run_sparql(self, query) -> QueryResult:
         """ Runs and returns the results from a sparql query"""
         self.sparql.setQuery(query)
-        self.sparql.setReturnFormat(JSON)
+        self.sparql.setReturnFormat("json")
         return self.sparql.query()
     
-    def get_datasets(self):
+    def map_query_response_to_json(self, list_of_data):
+        nicer_list = []
+        for item in list_of_data:
+            n = {
+                "title": get_value_from_dict(item, "name"),
+                "description": get_value_from_dict(item, "description"),
+                "summary": get_value_from_dict(item, "comment"),
+                "last_updated": get_value_from_dict(item, "modified"),
+                "links": {"self": {"url": "Mike will get the value from Flask"},
+                "publisher": {"url": get_value_from_dict(item, "creator"),
+                              "id": get_value_from_dict(item, "creatorName")},
+                "topic": {"url": get_value_from_dict(item, "theme"),
+                          "id": get_value_from_dict(item, "themeName")},
+                "releases":{"url": get_value_from_dict(item, "comment")},
```

Comment:
as discussed, you'll need to check in with @eldeal on this. "releases" and "latest_version" as these distinctions are not (as far as I know) implemented on the main PMD yet.

for the moment every dataset will be its own latest version so you can just use the id for the dataset you're looking at (see other comment) and the same "url" you're building for self.

"releases" I don't know, you'll need to check in with Eleanor

Reply:
agree, latest version is the same as self for now. but yeah we can remove releases, we're far from that existing